### PR TITLE
 [BISERVER-14720] Upgrade log4j2

### PR DESCRIPTION
**Pull request by rmansoor**

### CODE REVIEW

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 7m
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl \
assemblies/core/static,assemblies/static,plugins/pur/core
```
---
 :exclamation: No tests found!
 ---
 
 

:information_source:  _This is an automatic message_

File: plugins/pur/core/src/test/resources/log4j2.xml [hunk 1]
```diff
@@ -16,6 +16,7 @@
       <!-- Append messages to the console -->
       <!-- ============================== -->
       <Console name="PENTAHOCONSOLE">
+         <ThresholdFilter value="INFO"/>
```
Comment (bennychow) [inline plugins/pur/core/src/test/resources/log4j2.xml:19]: I looked at the history of the corresponding log4j.xml and it didn't have this threshold set.

https://github.com/pentaho/pentaho-kettle/blob/9.2.0.2/plugins/pur/core/src/test/resources/log4j.xml

**Comment by rmansoor** [comment]
@bennychow please review, I fixed it

**Review by bennychow — APPROVED** [review]

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 3m 24s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl \
assemblies/core/static,assemblies/static
```
---
 :exclamation: No tests found!
 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 2m 59s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl \
assemblies/core/static,assemblies/static
```
---
 :exclamation: No tests found!
 ---
 
 

:information_source:  _This is an automatic message_

**Review by bennychow — APPROVED** [review]

**Pull request merged**